### PR TITLE
LunaX audit fixes 2

### PR DESCRIPTION
**Pull request by bharath-123**

### CODE REVIEW

File: contracts/staking/src/helpers.rs [hunk 1]
```diff
@@ -25,6 +25,17 @@ pub enum Verify {
...
 
+pub fn validate_unbonding_period(unbonding_period: u64) -> bool {
+    // unbonding period should be in [21 days, 21 days + 30mins]
+    unbonding_period < 1816200 && unbonding_period >= 1814400
+}
+
+pub fn validate_undelegation_cooldown(undelegation_cooldown: u64) -> bool {
+    // undelegation cooldown should be in [3 days - 10mins, 3 days)
+    println!("undelegation cooldown is {:?}", undelegation_cooldown);
+    undelegation_cooldown <= 259200 && undelegation_cooldown >= 258600
```
Comment (galacticminter) [inline contracts/staking/src/helpers.rs]: Shouldn't undelegation cooldown be (3 days, 3 days 10min)?

Comment (bharath-123) [reply contracts/staking/src/helpers.rs]: This is for jobs. We usually want to allow undelegation cooldown to be a bit lower than 3 days. But I will increase the bounds

**Review by galacticminter — APPROVED** [review]

File: contracts/staking/src/contract.rs [hunk 2]
```diff
@@ -120,17 +137,20 @@ pub fn instantiate(
...
-        withdraw_paused: false,
-        reinvest_paused: false,
-        reconcile_paused: false,
-        claim_airdrops_paused: false,
-        redeem_rewards_paused: false
-    })?;
+
+    OPERATION_CONTROLS.save(
+        deps.storage,
+        &OperationControls {
```
Comment (batphonghan) [inline contracts/staking/src/contract.rs:143]: Since all the pros is `false` I think we can drive (Default) then do : OperationControls::default().

**Review by batphonghan — APPROVED** [review]

File: contracts/staking/src/contract.rs (hunk 2 above)
Comment (bharath-123) [reply contracts/staking/src/contract.rs:143]: good suggestion

**Pull request merged**